README.Rmd

```diff
@@ -125,6 +125,53 @@ mins_to_hours <- function(data) {
 ark(db, dir, lines = 50000, tables = "flights", callback = mins_to_hours)
 ```
 
+## ark() in parallel
+
+There are two strategies for using `ark` in parallel. One is to loop over the tables, re-using the ark function per table in parallel. The other, introduced in 0.0.15, is to use the "window-parallel" method. This is particularly useful if your tables are very large and can speed up the process significantly. 
+
+```{r, eval = FALSE}
+# Strategy 1: Parallel over tables
+library(arkdb)
+library(future.apply)
+
+plan(multisession)
+
+future_lapply(vector_of_tables, function(x) ark(db, dir, lines, tables = x))
```

Comment:
this one seems to use the default `streamable_table`, but the document doesn't mention that these other strategies are also changing the choice of output format.

Reply:
I made some updates to make this more explicit, and clear.